lib/aws-serverless-textract-solution-stack.ts

```diff
@@ -9,5 +13,18 @@ export class AwsServerlessTextractSolutionStack extends cdk.Stack {
     if (!props.resourcePrefix || !props.cdkDeployEnvironment || !props.appName) {
       throw new Error('Missing required properties in CxObPrivateLlmProviderStackProps.');
     }
+
+    // check s3 bucket names
+    checkEnvVariables('MASTER_BUCKET_NAME', 'PFDFILES_BUCKET_NAME', 'JSONFILES_BUCKET');
+
+    // create s3 buckets stack
+    const awsServerlessS3StackProps: AwsServerlessS3StackProps = {
+      ...props,
+      masterBucketName: process.env.MASTER_BUCKET_NAME!,
+      pfdFilesBucketName: process.env.PFDFILES_BUCKET_NAME!,
+      jsonFilesBucketName: process.env.JSONFILES_BUCKET_NAME!,
+      encryption: s3.BucketEncryption.S3_MANAGED,
+    };
```

Comment:
**Suggestion:** Ensure that the environment variables `MASTER_BUCKET_NAME`, `PFDFILES_BUCKET_NAME`, and `JSONFILES_BUCKET` are defined before accessing them to avoid potential runtime errors.
> 🥇  **High Priority** 
 > [Category: **`Possible Issue`**, Importance: **`9`**]
> **Reason:** This suggestion addresses a potential runtime error by ensuring that critical environment variables are defined before they are accessed, enhancing the robustness of the code.

```suggestion
    checkEnvVariables('MASTER_BUCKET_NAME', 'PFDFILES_BUCKET_NAME', 'JSONFILES_BUCKET');

    if (!process.env.MASTER_BUCKET_NAME || !process.env.PFDFILES_BUCKET_NAME || !process.env.JSONFILES_BUCKET) {
      throw new Error('One or more required environment variables are missing.');
    }

    const awsServerlessS3StackProps: AwsServerlessS3StackProps = {
      ...props,
      masterBucketName: process.env.MASTER_BUCKET_NAME,
      pfdFilesBucketName: process.env.PFDFILES_BUCKET_NAME,
      jsonFilesBucketName: process.env.JSONFILES_BUCKET,
      encryption: s3.BucketEncryption.S3_MANAGED,
    };
```
- [ ]  **I have reviewed this code suggestion and addressed the relevant ones.**